docs/en/stack/ml/anomaly-detection/ml-troubleshooting.asciidoc

```diff
@@ -8,6 +8,42 @@
 Use the information in this section to troubleshoot common problems and known
 issues.
 
+[discrete]
+[[ml-avoid-upgrade-closures]]
+== Avoiding spurious {anomaly-job} closures on upgrade
```

Comment:
> == Avoiding spurious {anomaly-job} closures on upgrade

I am uncertain which meaning of spurious is intended here. Could we simplify with a word like "unnecessary" or "problematic"?

Reply:
How about "unintended"?

Reply:
That's great, I'll make that change here.